sql/moz-fx-data-shared-prod/telemetry_derived/clients_histogram_bucket_counts_v1/query.sql

```diff
@@ -18,12 +18,11 @@ WITH filtered_data AS (
     os = 'Windows'
     AND channel = 'release' AS sampled
   FROM
-    clients_histogram_aggregates_v1
+    clients_histogram_aggregates_v2
   CROSS JOIN
     UNNEST(histogram_aggregates)
   WHERE
-    submission_date = @submission_date
-    AND first_bucket IS NOT NULL
+    first_bucket IS NOT NULL
```

Comment:
why no more submission_date?

Reply:
`clients_histogram_aggregates_v1` used to hold 7 days of data and was partitioned by submission date, therefore this was required. With `clients_histogram_aggregates_v2` we only have one day of data (corresponding to the last execution date) so we always want to query the whole table

Reply:
thanks for the explanation! let's document this somewhere